vaadin/web-components
# fix: handle moving CRUD editor properly

## Description

Fixes vaadin/flow-components#1794

## Type of change


## Note

1. Updated `@web/test-runner` to the latest version with Playwright 1.12 affected by WebKit issue,
2. Refactored logic to avoid teleporting `<slot name="form">` and move the actual form instead.

## How to test

1. Run `yarn debug --group vaadin-crud`
2. Press M for manual mode
3. Open the link in Safari
4. Check the new test file `crud-editor.test.js`
